todo-mvc/Gruntfile.js

```diff
@@ -1,45 +1,3 @@
 module.exports = function (grunt) {
-
-	var staticFiles = [ 'src/**/*.html', 'src/**/*.css' ];
-	var testFiles = [ 'tests/support/index.html' ];
-
-	require('grunt-dojo2').initConfig(grunt, {
-		ts: {
-			dist: {
-				compilerOptions: {
-					declaration: false
-				}
-			}
-		},
-		copy: {
-			staticFiles: {
-				expand: true,
-				cwd: '.',
-				src: staticFiles,
-				dest: '<%= devDirectory %>'
-			},
-			testIndexFiles: {
```

Comment:
we still need this copy config also (but the others can go if as far as I can remember)